parsers/s01-parse/dawstudiocenter/openvpnddwrt-logs.yaml

```diff
@@ -0,0 +1,29 @@
+name: dawstudiocenter/openvpnddwrt-logs
+description: "Parse ddwrt logs for openvpn messages"
+filter: "evt.Meta.machine == 'VPNRouter'"
```

Comment:
Not everyone machine will be named `VPNRouter` ?

Reply:
OK I understand now. You're right. I filter syslog message using my router name.
I need to remove this filter condition or adapt it.
Let me think about it .